bin/get_tokens

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "base64"
+require 'dotenv'
+require 'httparty'
+
+Dotenv.load('.env.test')
```

Comment:
Doesn't dotenv load files in a specific order, so that `.ev.test` would override any `.env` that was there ...
Also the `.env.test` file should be committed to the repo. I seem to remember from above that it was added to `.gitignore`.

Reply:
I've changed this script now. Please have a second look at it.